src/connectors/metrics.rs

```diff
@@ -51,7 +51,11 @@ pub(crate) struct MetricsChannel {
 
 impl MetricsChannel {
     pub(crate) fn new(qsize: usize) -> Self {
-        let (tx, rx) = bounded(qsize);
+        let (mut tx, rx) = broadcast(qsize);
+        // We user overflow so that non collected messages can be removed
+        // FIXME: is this what we want? for Metrics it should be good enough
```

Comment:
What would be ideal here would be that older messages get discarded, not newer ones. If we have this measure in place, and if sending will thus never wait for the channel being empty we can remove the hack with the keep_alive flag for sources

Reply:
Investigating the async-broadcast crate, this is exactly what happens :D I misread it the first time:

https://docs.rs/async-broadcast/0.3.3/async_broadcast/struct.Receiver.html#method.set_overflow

Reply:
I will take care of removing the superfuous `keep_alive` stuff that i introduced for the metrics connector then. :)

Reply:
:rocket: